contrib/endpoints/src/api_manager/config_test.cc

```diff
@@ -870,6 +870,86 @@ TEST(Config, TestCorsDisabled) {
   ASSERT_EQ(nullptr, method1);
 }
 
+TEST(Config, TestInvalidMetricRules) {
+  MockApiManagerEnvironmentWithLog env;
+  static const char config_text[] = R"(
```

Comment:
Please add comment to state why this is invalid
// There is no http.rule or api.method to match the selector.